identifiers/country-us/state-nc-local_gov.csv

```diff
@@ -27,9 +27,9 @@ ocd-division/country:us/state:nc/county:chatham/council_district:2,Chatham Count
 ocd-division/country:us/state:nc/county:chatham/council_district:3,Chatham County District 3
 ocd-division/country:us/state:nc/county:chatham/council_district:4,Chatham County District 4
 ocd-division/country:us/state:nc/county:chatham/council_district:5,Chatham County District 5
-ocd-division/country:us/state:nc/county:cherokee/council_district:1,Cherokee County District 1
-ocd-division/country:us/state:nc/county:cherokee/council_district:2,Cherokee County District 2
-ocd-division/country:us/state:nc/county:cherokee/council_district:3,Cherokee County District 3
+ocd-division/country:us/state:nc/county:cherokee/council_district:1,Cherokee County Council District 1
```

Comment:
I don't understand why the word 'Council' needs to be added. It's not consistent with the names of existing council districts.